src/Models/Constants/Interfaces.ts

```diff
@@ -733,32 +733,36 @@ export interface IADT3DViewerProps extends BaseComponentProps {
     sceneId: string;
     scenesConfig: I3DScenesConfig;
     pollingInterval: number;
-    title?: string;
-    connectionLineColor?: string;
-    enableMeshSelection?: boolean;
+    externalProps?: ADT3DViewerExternalProps;
     addInProps?: IADT3DAddInProps;
     refetchConfig?: () => any;
-    showMeshesOnHover?: boolean;
-    showHoverOnSelected?: boolean;
-    coloredMeshItems?: CustomMeshItem[];
-    /**
-     * Ids of the elements to zoom the camera to focus on
-     */
-    zoomToElementId?: string;
-    unzoomedMeshOpacity?: number;
-    hideViewModePickerUI?: boolean;
-    hideElementsPanel?: boolean;
-    outlinedMeshItems?: CustomMeshItem[];
     /** show the toggle to switch between builder & viewer modes */
     showModeToggle?: boolean;
     sceneViewProps?: ISceneViewProps;
-    selectedLayerIds?: string[];
     /**
      * Call to provide customized styling that will layer on top of the variant rules.
      */
     styles?: IStyleFunctionOrObject<IADT3DViewerStyleProps, IADT3DViewerStyles>;
 }
 
+export interface ADT3DViewerExternalProps {
```

Comment:
"ExternalProps" is an odd name to see as a consumer, I wouldn't know what to expect inside of that object or how it is going to be used. Can we think about a more intuitive name? 
For the props like "hideElementsPanel" I liked the idea of `settings` or `viewerSettings` or `settingsOverride`.
For the props like connectionLineColor, let's think about a name that fits that collection.

Reply:
Are you suggesting several groups of props?